packages/features/bookings/Booker/Booker.tsx

```diff
@@ -291,7 +291,9 @@ const BookerComponent = ({
                   (layout === BookerLayouts.COLUMN_VIEW || layout === BookerLayouts.WEEK_VIEW) &&
                     "bg-default dark:bg-muted sticky top-0 z-10"
                 )}>
-                {!isPlatform ? (
+                {isPlatform && layout === BookerLayouts.MONTH_VIEW ? (
+                  <></>
+                ) : (
                   <Header
```

Comment:
<img width="282" alt="Screenshot 2024-09-11 at 3 49 38 PM" src="https://github.com/user-attachments/assets/95f54b74-0faa-49e8-bbe5-962371fd4787">

For monthly view the above header is a sticky component and would look weird since it'll overflow any object on top of it, thats why its better to hide it for monthly view